src/pretalx/submission/models/submission.py

```diff
@@ -124,6 +124,14 @@ class Submission(LogMixin, models.Model):
     notes = models.TextField(
         null=True, blank=True, verbose_name=_('Notes for the organiser')
     )
+    orga_notes = models.TextField(
```

Comment:
Please call the field internal_notes instead.

Reply:
The `notes` field is currently not exposed by the API as well. Shall I add it?